src/client/src/rt-theme/tools.ts

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash'
+
+export const resolvesColor = (color, other) => {
```

Comment:
Are you even using this?

Reply:
No, not at the moment, casualty of copy paste — will remove.